sdk/highlight-node/src/sdk.ts

```diff
@@ -84,6 +85,30 @@ export const H: HighlightInterface = {
 			console.warn('highlight-node flush error: ', e)
 		}
 	},
+	log: (
+		message: any,
+		level: string,
+		secureSessionId?: string | undefined,
+		requestId?: string | undefined,
+	) => {
+		const o: { stack: any } = { stack: {} }
+		const saveTraceLimit = Error.stackTraceLimit
+		Error.stackTraceLimit = 3
```

Comment:
Out of curiosity, why 3?

Reply:
good catch - we shouldn't be doing a limit on the sdk side but rather limit how we render long frames if necessary on the frontend. i'll remove this here.